mono/mini/mini-x86.h

```diff
@@ -68,9 +68,11 @@ LONG CALLBACK seh_handler(EXCEPTION_POINTERS* ep);
 #define MONO_ARCH_SUPPORT_TASKLETS 1
 
 #ifndef DISABLE_SIMD
+#ifndef ENABLE_NETCORE
 #define MONO_ARCH_SIMD_INTRINSICS 1
```

Comment:
This disables something which used to work. Not sure it matters through.

Reply:
Oh right, I forgot about it at the draft stage. At this point I don't think there's any value in porting the lowering code to x86 so we can either live with disabling it (no one tests it anyway), or I will have to disable it selectively for non-LLVM case.

Reply:
It was already a bit broken before because we always reported HW support for `Vector<T>` even on CPUs with no SSE(2) instructions. LLVM probably handles it somehow and generates software fallbacks but it is untested code path.